docs/migrating_to_nextcord.rst

```diff
@@ -8,3 +8,43 @@ Migrating to nextcord
 Due to the `original discord.py repository <https://github.com/Rapptz/discord.py>`_ becoming read-only, we decided
 that it would be necessary to fork it and keep on developing further. We also wanted to change the name and voted on
 nextcord in order to properly register it at pypi.
+
+Porting from discord.py
+-------------------------
+
+In order to port a bot using discord.py to nextcord, follow these steps:
+
+1. Install nextcord: 
+
+.. code:: sh
+
+    # Linux/macOS
+    python3 -m pip install -U nextcord
+
+    # Windows
+    py -3 -m pip install -U nextcord
+
+2. Uninstall discord.py:
+
+..code:: sh
+
+    # Linux/macOS
+    python3 -m pip uninstall discord
+
+    # Windows
+    py -3 -m pip uninstall discord
+
+3. Update the following import statements:
+
+* ``import discord`` -> ``import nextcord as discord``

```

Comment:
Instead of `import nextcord as discord` it could be `import nextcord`

Reply:
Import as means they don't have to do much code change

Reply:
I think this should be changed. It's important that people switch to using the correct library name, we can't hijack the `discord` name forever, and it will become difficult to help people since 3 libraries now do this. How would you know if someone is referring to discord.py, pycord, or nextcord?

Reply:
Okay, I changed this and added a fourth step for them to change their code to use `nextcord` instead of `discord`.